### DISCUSSION TITLE
Make HowToClaimRegex equals TrappedChatKeyword

### DISCUSSION THREAD

**Discussion by SrBedrock** [GriefPrevention/GriefPrevention#1144] [Ideas]
As admin and player I would like the `HowToClaimRegex` option to work similar to the `TrappedChatKeyword` option.

When the player sends words like claim, protect or lock in chat nothing happens.
When players send in chat words defined in `TrappedChatKeyword` work as they should.

**Comment by RoboMWM** [comment]
this is a dupe of that other one yes or is it a brand new suggestion

**Reply by SrBedrock** [reply]
Brand new suggestion!
